src/scenes/PlantsDashboardRouter/components/LiveDeadPlantsPerSpeciesCard.tsx

```diff
@@ -33,21 +33,27 @@ export default function LiveDeadPlantsPerSpeciesCard(): JSX.Element {
 
   useEffect(() => {
     if (observationSummaries?.[0]) {
+      const filterFn = isSurvivalRateCalculationEnabled
+        ? (sp: any) => sp.survivalRate !== undefined && sp.survivalRate !== null
+        : (sp: any) => sp.cumulativeDead !== 0 || sp.permanentLive !== 0;
+
       const speciesNames = observationSummaries[0].species
-        .filter((sp) => sp.cumulativeDead !== 0 || sp.permanentLive !== 0)
+        .filter(filterFn)
         .map((sp) => ({
           label:
             availableSpecies.find((availableS) => availableS.id === sp.speciesId)?.scientificName ||
             sp.speciesName ||
             '',
           value: sp.speciesId?.toString() || '',
-        }));
+        }))
+        .sort((a, b) => a.label.localeCompare(b.label));
```

Comment:
```suggestion
        .sort((a, b) => a.label.localeCompare(b.label, activeLocale || undefined));
```

I think we should pass in the locale.